src/components/date-range/date-range.test.js

```diff
@@ -0,0 +1,437 @@
+      CypressMountWithProviders(<DateRangeCustom endWarning />);
+
+      checkInputColor(END_DATE_RANGE_INDEX, WARN_COLOUR);
+    });
+
+    it("should check the startInfo as boolean", () => {
+      CypressMountWithProviders(<DateRangeCustom startInfo />);
+
+      checkInputColor(START_DATE_RANGE_INDEX, INFO_COLOUR);
+    });
+
+    it("should check the endInfo as boolean", () => {
+      CypressMountWithProviders(<DateRangeCustom endInfo />);
+
+      checkInputColor(END_DATE_RANGE_INDEX, INFO_COLOUR);
+    });
+
+    it.each(testData)(
+      "should check the validationOnLabel with error state",
+      (error) => {
+        CypressMountWithProviders(
+          <DateRangeCustom
+            startError={error}
+            endError={error}
+            validationOnLabel
+          />
+        );
+
+        dateRangeComponentLabel(START_DATE_RANGE_INDEX)
+          .parent()
+          .find(ERROR_ICON)
+          .realHover();
+        getDataElementByValue("tooltip").should("have.text", error);
+        dateRangeComponentLabel(END_DATE_RANGE_INDEX)
+          .parent()
+          .find(ERROR_ICON)
+          .realHover();
+        getDataElementByValue("tooltip").should("have.text", error);
+      }
+    );
+
+    it.each(testData)(
+      "should check the validationOnLabel with warning state",
+      (warning) => {
+        CypressMountWithProviders(
+          <DateRangeCustom
+            startWarning={warning}
+            endWarning={warning}
+            validationOnLabel
+          />
+        );
+
+        dateRangeComponentLabel(START_DATE_RANGE_INDEX)
+          .parent()
+          .find(WARNING_ICON)
+          .realHover();
+        getDataElementByValue("tooltip").should("have.text", warning);
+
+        dateRangeComponentLabel(END_DATE_RANGE_INDEX)
+          .parent()
+          .find(WARNING_ICON)
+          .realHover();
+        getDataElementByValue("tooltip").should("have.text", warning);
+      }
+    );
+
+    it.each(testData)(
+      "should check the validationOnLabel with info state",
+      (info) => {
+        CypressMountWithProviders(
+          <DateRangeCustom startInfo={info} endInfo={info} validationOnLabel />
+        );
+
+        dateRangeComponentLabel(START_DATE_RANGE_INDEX)
+          .parent()
+          .find(INFO_ICON)
+          .realHover();
+        getDataElementByValue("tooltip").should("have.text", info);
+
+        dateRangeComponentLabel(END_DATE_RANGE_INDEX)
+          .parent()
+          .find(INFO_ICON)
+          .realHover();
+        getDataElementByValue("tooltip").should("have.text", info);
+      }
+    );
+
+    it.each([
+      [true, "top"],
+      [false, "bottom"],
+    ])(
+      "should check the labelsInline prop is set to %s",
+      (boolean, cssValue) => {
+        CypressMountWithProviders(<DateRangeCustom labelsInline={boolean} />);
+
+        dateRange(START_DATE_RANGE_INDEX).should(
+          "have.css",
+          "vertical-align",
```

Comment:
It might be better to check `"display"`, `"flex"` is on the DateInput container here as that is the css that is making it inline the labels

<img width="1478" alt="image" src="https://user-images.githubusercontent.com/44157880/175516892-efda396e-54ee-47c5-a3fe-32ab7c745b99.png">

Reply:
thx @edleeks87, you are right :) fixed